install_config/topics/optional_configuration_for_rhv.adoc

```diff
@@ -0,0 +1,71 @@
+////
+Module included in the following assemblies:
+
+install_config/configuring_for_rhv.adoc
+////
+[id="optional_configuration_for_rhv"]
+= Optional configuration for Red Hat Virtualization
+
+To change the {product-title} configuration for Red Hat Virtualization, modify the following parameter values in the Ansible inventory file before installing the cluster:
```

Comment:
@rgolangh @jan-zmeskal 

Why do we need to have a separate section for this inventory file? Wouldn't it make more sense to put these sections in the openshift_3_11.hosts file with callouts indicating the options?

Reply:
@rgolangh @jan-zmeskal @shlao 

It looks like this 'optional' inventory file is left over from old documentation: https://docs.openshift.com/container-platform/3.11/install_config/configuring_rhv.html#configuring-openshift-container-platform-for-red-hat-virtualization

Can we get rid of this section?